htdocs/class/xml/rpc/xmlrpcapi.php

```diff
@@ -49,14 +49,14 @@ class XoopsXmlRpcApi
     protected $isadmin = false;
 
 
-    function XoopsXmlRpcApi(array &$params, XoopsXmlRpcResponse &$response, XoopsModule &$module)
+    function XoopsXmlRpcApi(array &$params, XoopsXmlRpcResponse response, XoopsModule &$module)
```

Comment:
Missing '$'.  Also, why leave the & on module? It is an object, too, so the reference should be pointless?